.github/workflows/shared-run-e2e-tests.yaml

```diff
@@ -133,16 +130,37 @@ jobs:
       - name: Checkout uid2-shared-actions repo
         uses: actions/checkout@v3
         with:
-          ref: v2.7
+          ref: kcc-UID2-2339-create-test-pipeline-for-gcp
           repository: IABTechLab/uid2-shared-actions
           path: uid2-shared-actions
 
       - name: Checkout uid2-e2e repo
         uses: actions/checkout@v3
         with:
+          ref: kcc-UID2-2339-create-test-pipeline-for-gcp
           repository: IABTechLab/uid2-e2e
           path: uid2-e2e
 
+      - name: Create docker network
+        run: |
+          docker network create e2e_default
+
+      - name: Bring up bore
+        id: bore
+        if: ${{ inputs.operator_type == 'gcp' || inputs.operator_type == 'azure' }}
+        run: |
+          cd e2e && bash ../uid2-shared-actions/scripts/setup_bore.sh
+
+      - name: Prepare GCP metadata
+        id: prepare_gcp_metadata
+        if: ${{ inputs.operator_type == 'gcp'}}
+        uses: IABTechLab/uid2-shared-actions/actions/prepare_gcp_metadata@kcc-UID2-2339-create-test-pipeline-for-gcp
```

Comment:
I will update the branch name to v2 once https://github.com/IABTechLab/uid2-e2e/pull/6 is merged